ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/behavior/actions/RDXFootstepPlanAction.java

```diff
@@ -150,15 +149,15 @@ public RDXFootstepPlanAction(long id,
                                                            definition.getPlannerWalkWithGoalOrientation()::setValue,
                                                            imBoolean -> ImGui.checkbox(labels.get("Walk with goal orientation"), imBoolean));
       plannerParametersWidgets = new RDXStoredPropertySetTuner("Planner Parameters");
-      plannerParametersWidgets.create(definition.accessPlannerParameters(), false);
+      plannerParametersWidgets.create(definition.getPlannerParametersUnsafe(), false);
```

Comment:
Places like here need to access the values without freezing, otherwise, replication doesn't work. You don't want new replicated nodes starting out frozen.